src/HearThis/UI/DiscontiguousProgressTrackBar.cs

```diff
@@ -139,7 +139,7 @@ protected override void OnPaint(PaintEventArgs e)
 			//draw the bar
 			e.Graphics.FillRectangle(AppPallette.DisabledBrush, kLeftMargin, top, BarWidth, 3);
 
-			var brushes = GetSegmentBrushesOrErrorBrushes();
+			var brushes = GetSegmentBrushesMethod();
```

Comment:
It's basically a delegate (though it doesn't use the "delegate" keyword). I probably would have just called it GetSegmentBrushes, but I guess someone was hoping to try to make it more obvious that it's not actually a method implemented in this class. By doing it this way, it makes it possible for it to paint something in Designer that sort of illustrates how it could look when live. (Unfortunately, the method name for the version used in Designer (GetSegmentBrushesTest, which in turn calls GetSegsTest) is even worse.